polyply/src/gen_itp.py

```diff
@@ -102,6 +102,12 @@ def gen_params(name="polymer", outpath=Path("polymer.itp"), inpath=[], lib=None,
     LOGGER.info("applying links between residues",  type="step")
     meta_molecule = ApplyLinks().run_molecule(meta_molecule)
 
+    #Check missing edges and raise issues if links between amino acids are not applied.
+    #Bug if the backbone is correct links between amino acids in a chain are not applied correctly
+    for missing in find_missing_edges(meta_molecule, meta_molecule.molecule):
+        msg = "Link between residue {idxA} {resA} and residue {idxB} {resB} was not applied."
+        LOGGER.warning(msg, **missing)
```

Comment:
Instead of just adding this line we should check if there are missing edges and then raise a general error message like the "disconnected molecule warning" and then proceed to list the edges. The connected molecule check becomes then redundant and can be removed.